### ISSUE TITLE
Include ("hardlink") step from another recipe

### ISSUE THREAD

**Issue by HarHarLinks** [TandoorRecipes/recipes#1963]
### Is your feature request related to a problem? Please describe.

E.g. I already have a pizza recipe and the first step is about making the dough.
Now I am adding another recipe that reuses the same dough, such as pizza bread.

### Describe the solution you'd like

I would like to include the already existing step from another recipe.
It should include a scale factor for the amount of ingredients (e.g. three loaves of pizza bread need the same amount as two pizzas).
Internally, a reference to the existing step should get inserted into the database instead of simply copying it.

If I edit and save this step, it should ask me weather I want to modify the original step and remain linked, or instead instantiate a copy and unlink from the original.

### Describe alternatives you've considered

Just plain copy steps.

**Comment by tomtjes** [comment]
Another alternative would be to create a recipe "pizza dough" and add it as a step in both the "pizza" and the "pizza bread" recipe.

**Comment by HarHarLinks** [comment]
Right, good point. Just to be clear though, I would like not to just link to another recipe (which is already possible today) but properly integrate as one (or more) steps, ingredients list, etc.

**Comment by vabene1111** [comment]
it is planned to be able to re-use steps across recipes, most of what is needed for that is also already possible, just need to implement some kind of search for that

**Comment by RonnyAL** [comment]
Commenting to express support for this suggestion. Has any progress been made?
